src/lightkurve/io/cdips.py

```diff
@@ -63,6 +69,11 @@ def read_cdips_lightcurve(filename,
                                  quality_column=quality_column,
                                  time_format='btjd')
 
+    #The time displayed is in BJD not BTJD. We can fix this by subtracting
+    # 2457000
+    lc["time"] = lc["time"]-2457000
```

Comment:
This should be taken care of with the `time_format` argument by setting it to `jd`

Reply:
This has now been adjusted. Thank you.

Reply:
Note however that the time is given in BJD, but this does not seem to be an option.